cev_eris.dme

```diff
@@ -1627,6 +1628,32 @@
 #include "code\modules\clothing\under\jobs\medsci.dm"
 #include "code\modules\clothing\under\jobs\security.dm"
 #include "code\modules\clothing\underwear\base.dm"
+#include "code\modules\cooking_with_jane\cooking.dm"
+#include "code\modules\cooking_with_jane\cooking_catalog.dm"
+#include "code\modules\cooking_with_jane\cooking_init.dm"
+#include "code\modules\cooking_with_jane\cooking_tracker.dm"
+#include "code\modules\cooking_with_jane\food_overrides.dm"
+#include "code\modules\cooking_with_jane\step_defines.dm"
+#include "code\modules\cooking_with_jane\cooking_appliances\appliance_defines.dm"
+#include "code\modules\cooking_with_jane\cooking_appliances\cooking_circuits.dm"
+#include "code\modules\cooking_with_jane\cooking_appliances\grill.dm"
+#include "code\modules\cooking_with_jane\cooking_appliances\machinery.dm"
+#include "code\modules\cooking_with_jane\cooking_appliances\oven.dm"
```

Comment:
These 10 lines are good, will check in tommorow.

Reply:
I can't wait to merge this in 1.8 years